packages/paste-website/src/components/site-wrapper/site-header/HamburgerToggle.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import {Box, BoxProps} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+
+interface HamburgerToggleProps {
+  toggled: boolean;
+  onToggle: (newToggled: boolean) => void;
+}
+
+const Pattie: React.FC<BoxProps> = props => (
+  <Box
+    position="absolute"
+    width="12px"
+    height="2px"
+    borderRadius="borderRadius30"
+    transition="all .25s ease-in-out"
+    transformOrigin="left center"
+    {...props}
+  />
+);
+
+const HamburgerToggle = React.forwardRef<HTMLButtonElement, HamburgerToggleProps>(({toggled, onToggle}, ref) => {
+  const backgroundColor = toggled ? 'colorBackgroundBodyInverse' : 'colorBackgroundBody';
+  return (
+    <Button ref={ref} variant="reset" size="reset" onClick={() => onToggle(!toggled)}>
```

Comment:
For the button in the masthead, should it not be using our inverse button? https://twilio-labs.github.io/paste/?path=/story/components-button--inverse-button

Reply:
Designs currently look like this 
![image](https://user-images.githubusercontent.com/1592327/99535149-7f169c00-2976-11eb-92ef-7d1b82a1894d.png)

@corinne-nagel Want to swap to those?